src/main/java/uk/gov/pay/webhooks/app/WebhooksApp.java

```diff
@@ -54,7 +51,7 @@ public static void main(String[] args) throws Exception {
     }
     
     private static final String SERVICE_METRICS_NODE = "webhooks";
-    private static final int GRAPHITE_SENDING_PERIOD_SECONDS = 10;
+    private static final int METRICS_COLLECTION_PERIOD_SECONDS = 10;
```

Comment:
Should this be 30 similar to other app changes? https://github.com/alphagov/pay-products/pull/1210/files#diff-4f6066de42bd7de905cdc3cdfa57b9396cfa960c9ebea66f2559f7c21e74f6caR55

Reply:
Yeah it should! Thanks

Reply:
Updated 👍